File: src/RealtimeServer/scriptureforge/models/note-tag.ts
export const BIBLICAL_TERM_TAG_ICON = 'biblicalterm1';
export const DEFAULT_TAG_ICON = '01flag1';
export const SF_TAG_ICON = '06star2';
export const SF_TAG_NAME = 'Scripture Forge Note';
export const TO_DO_TAG_ID = 1;

export interface NoteTag {
  tagId: number;
  name: string;
  icon: string;
  creatorResolve: boolean;
}

File: src/SIL.XForge.Scripture/Models/NoteTag.cs
namespace SIL.XForge.Scripture.Models;

public class NoteTag
{
    public const string defaultTagIcon = "01flag1";
    public const string sfNoteTagIcon = "06star2";
    public const string sfNoteTagName = "Scripture Forge Note";
    public const int notSetId = 0;
    public int TagId { get; set; }
    public string Icon { get; set; }
    public string Name { get; set; }
    public bool CreatorResolve { get; set; }
}
